packages/cloud-cognitive/src/components/EditTearsheetNarrow/EditTearsheetNarrow.stories.js

```diff
@@ -0,0 +1,293 @@
+/**
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useState } from 'react';
+// TODO: import action to handle events if required.
+import { action } from '@storybook/addon-actions';
+import {
+  Button,
+  Dropdown,
+  NumberInput,
+  TextInput,
+  FormGroup,
+} from 'carbon-components-react';
+import { pkg } from '../../settings';
+import {
+  getStoryTitle,
+  prepareStory,
+} from '../../global/js/utils/story-helper';
+
+import { EditTearsheetNarrow } from '.';
+import { CreateTearsheetNarrow } from '../CreateTearsheetNarrow';
```

Comment:
@elycheea Since we are referring from CreateTearsheetNarrow rather than creating a new component for EditTearsheetNarrow, we need this import to use as below

<img width="548" alt="image" src="https://user-images.githubusercontent.com/29351394/167810893-ab2b02a0-ca19-416b-99ba-708a58e119c2.png">